Composer/packages/client/src/pages/language-generation/index.tsx

```diff
@@ -113,15 +126,15 @@ const LGPage: React.FC<RouteComponentProps> = props => {
 
   function onSelect(id) {
     if (id === '_all') {
-      navigateTo('/language-generation');
+      navigateTo('/language-generation/');
     } else {
       navigateTo(`language-generation/${id}`);
     }
   }
 
   function onToggleEditMode() {
     setEditMode(!editMode);
-    setCodeRange(null);
+    setCodeRange(undefined);
```

Comment:
I am unsure if `useState` allow empty args for setter, but it will be cleaner if we can use `setCodeRange()` instead.

Reply:
I find that the setCodeRange doesn't support empty args.